tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -255,10 +259,14 @@ public function expirationDateChangedTo($type, $receiver, $days) {
 	 * @param string $receiver
 	 *
 	 * @return void
+	 * @throws Exception
 	 */
 	public function expirationDateShouldBe($days, $type, $receiver) {
-		$expectedDate = \date('d-m-Y', \strtotime($days));
-		Assert::assertEquals($this->sharingDialog->getExpirationDateFor($receiver, $type), $expectedDate);
+		if (\strtotime($days) !== false) {
+			Assert::assertEquals(\strtotime($days), $this->sharingDialog->getExpirationDateFor($receiver, $type));
```

Comment:
```suggestion
			Assert::assertEquals(\date(\strtotime($days)), $this->sharingDialog->getExpirationDateFor($receiver, $type));
```

Probably, also format might need to be passed as `Y-m-d`.